src/AzureAuth/Ado/Constants.cs

```diff
@@ -26,7 +26,8 @@ public static class Client
         public static class Scope
         {
             /// <summary>
-            /// The default scope used for Azure Devops.
+            /// The default scope used for Azure DevOps.
+

```

Comment:
extra line.

Reply:
Something very odd is happening, because none of these warnings are reproing locally, and there's no extra line visible in the file when I look locally.

Reply:
Did you use the `code suggestion apply` in GitHub? It often added an extra line.
I always edit the suggestion code first if there is an extra line.

Reply:
Oh interesting....

Reply:
Thank you Rick! I had not pulled those commits yet. I'll have to test that later. I think you can craft the suggestion to not contain a newline. If the suggestion comment had a newline in it, it would get added yes.